docs/devices/MJWSD05MMC.md

```diff
@@ -1,12 +1,14 @@
-# Xiaomi MJWSD05MMC ATC or PVVX firmware
+# Xiaomi MJWSD05MMC ATC, PVVX or BTHome v2(*) firmware
 
-|Model Id|[MJWSD05MMC_ATC/PVVX](https://github.com/theengs/decoder/blob/development/src/devices/LYWSD03MMC_json.h)|
+|Model Id|[MJWSD05MMC_ATC/PVVX/BTHOME](https://github.com/theengs/decoder/blob/development/src/devices/LYWSD03MMC_json.h)|
 |-|-|
 |Brand|Xiaomi|
 |Model|Compact Temperature sensor|
 |Short Description|Temperature and humidity sensor also showing date and time with ATC or PVVX firmware|
 |Communication|BLE broadcast|
 |Frequency|2.4Ghz|
 |Power Source|CR2450|
-|Exchanged Data|temperature, humidity, battery, volt|
-|Encrypted|No|
+|Exchanged Data|temperature, humidity, battery, voltage, packet, power, open (depending on which MJWSD05MMC firmware is installed)|
+|Encrypted|Yes/No - Optional|
+
+(*) For the BTHome v2 format to be recognised the assigned name needs to start with "ATC" when flashing the firmware
```

Comment:
I would say:

For the BTHome v2 format to be recognised the assigned name must not be renamed from the default prefix of "ATC" when configuring the sensor via the PVVX TelinkMiFlasher

Or we take out the ATC condition?